corehq/apps/data_cleaning/models.py

```diff
@@ -309,10 +309,10 @@ def _attach_change_to_records(self, change, doc_ids=None):
         change.records.add(*selected_records)
 
     @transaction.atomic
-    def apply_change_to_selected_records_in_queryset(self, change):
+    def apply_change_to_selected_records(self, change):
```

Comment:
I think passing around an unsaved object can be a bit difficult to follow. What would you think of passing the `cleaning_form` into this function and having it handle both creating and saving the change?

Reply:
sure i can do a refactor in a future PR (doing a cleanup pass this upcoming sprint)